src/main/java/com/ksabbak/javaserver/app/controller/CatFormController.java

```diff
@@ -0,0 +1,34 @@
+package com.ksabbak.javaserver.app.controller;
+
+import com.ksabbak.javaserver.server.Response;
+import com.ksabbak.javaserver.server.StatusCode;
+import com.ksabbak.javaserver.storage.Persistable;
+
+import java.util.Map;
+
+public class CatFormController extends Controller {
+
+    @Override
+    public Response post(String params, Persistable storage) {
+        StatusCode statusCode = StatusCode.OK;
+
+        Map<String, String> formattedParams = super.stringToHashMap(params);
+
+        for(Map.Entry<String, String> pair : formattedParams.entrySet()) {
+            System.out.println(pair.getKey());
+            if (pair.getKey().equals("data")) {
+                try {
+                    storage.create(pair.getKey(), pair.getValue());
+                    statusCode = StatusCode.CREATED;
+                    return new Response.ResponseBuilder(statusCode).location("/cat-form/data").build();
+                } catch (Exception e) {
```

Comment:
Right now, this is just swallowing the exception, which might lead to future problems that are difficult to debug.

Reply:
Makes sense.